models/articles.models.js

```diff
@@ -1,20 +1,33 @@
 const db=require("../db/connection.js")
 
-exports.selectAllArticles=((articleQuery)=>{
-    //articleQuery= "articles." + articleQuery
-    //console.log(articleQuery)
+exports.selectAllArticles=((articleQuery, articleSortType)=>{
+    const allowedInputs=["title", "topic", "author", "body", "created_at", "ASC", "DESC", "votes", undefined]
```

Comment:
hmmmm I think I see why you've put undefined here but it would be much better to set your sort_by to have a default on line 3. That way, it will never be undefined (which as per the instructions on the ticket, it shouldn't be anyway)

Reply:
This variable also might be better named as something more descriptive of what they actually are, something like allowedSortBys